components/builder-web/app/origin/origin-page/origin-members-tab/_origin-members-tab.component.scss

```diff
@@ -50,14 +51,6 @@
           }
         }
       }
-
-      .members-pending-invites {
-        @include span-columns(4 of 9);
-
-        p {
-          font-size: rem(14);
-        }
-      }
     }
```

Comment:
The pending invites were lacking space to the right of the form to invite new users. Related, we started using multiple stacked lists in places (Packages view) so this re-uses that patter - putting Pending Invites in a list above Current Members.